src/server/routes/hack-challenges/add-hack-challenges.ts

```diff
@@ -5,24 +5,11 @@ import EventBroadcaster from '../../eventbroadcaster'
 import * as Boom from '../../boom'
 
 export default async function handler(req: Request, reply: IReply) {
-  const hackId = req.params.hackId
+  const { hackId: hackid } = req.params
```

Comment:
New javascript hurts to read in typescript. Not saying change it, just, it hurts.